source/userguide/auto_annotator.rst

```diff
@@ -0,0 +1,489 @@
+Working with the Auto Annotator
+=====================================================
+
+The Auto Annotator
+
+  - is a tool to automatically annotate or unannotate select entities across all labelled data in an application.
+  - supports the development of custom Annotators.
+
+.. note::
+
+   The examples in this section require the :doc:`HR Assistant <../blueprints/hr_assistant>` blueprint application. To get the app, open a terminal and run ``mindmeld blueprint hr_assistant``.
+
+.. warning::
+
+   Changes by an Auto Annotator cannot be undone and Mindmeld does not backup query data. We recommend using version control software such as Github.
+
+Quick Start
+-----------
+This section briefly explains the use of the unannotate and annotate commands. For more details, read the next section.
+
+First, copy and paste the config below into the :attr:`config.py` of the mindmeld app that you would like to remove annotations.
+
+.. code-block:: python
+
+	AUTO_ANNOTATOR_CONFIG = { 
+
+		"annotator_class": "SpacyAnnotator",
+		"overwrite": False, 
+		"annotate": [{"domains": ".*", "intents": ".*", "files": ".*", "entities": ".*"}],
+		"unannotate_supported_entities_only": True, 
+		"unannotate": [{"domains": ".*", "intents": ".*", "files": ".*", "entities": ".*"}],
+	}
+
+To unannotate you can run python code or use the command-line.
+
+Python:
+
+.. code-block:: python
+
+	from mindmeld.auto_annotator import SpacyAnnotator 
+	sa = SpacyAnnotator(app_path="hr_assistant")
+
+	sa.unannotate()
+
+Command-line:
```

Comment:
Let's show the Python command first

Reply:
The Python command for "annotate" and "unannotate" is shown before the CLI command

Unannotate
- Python
- CLI

Annotate
- Python
- CLI

Is there a different structure that you would prefer?